.github/actions/colab/colab-python-py37-pytorch/action.yml

```diff
@@ -52,7 +52,7 @@ runs:
         pip install -i https://pypi.org/simple --pre bigdl
         pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} torch==1.7.1 torchvision==0.8.2
         pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} six cloudpickle
```

Comment:
can we also delete these two dependencies?

Reply:
removed